COMPONENT_INDEX.md

```diff
@@ -351,10 +352,18 @@ export type BreakpointValue = 320 | 672 | 1056 | 1312 | 1584;
 
 ### Props
 
-| Prop name | Required | Kind             | Reactive | Type                                         | Default value                                                             | Description                                       |
-| :-------- | :------- | :--------------- | :------- | -------------------------------------------- | ------------------------------------------------------------------------- | ------------------------------------------------- |
-| sizes     | No       | <code>let</code> | Yes      | <code>Record<BreakpointSize, boolean></code> | <code>{ sm: false, md: false, lg: false, xlg: false, max: false, }</code> | Carbon grid sizes as an object                    |
-| size      | No       | <code>let</code> | Yes      | <code>BreakpointSize</code>                  | <code>undefined</code>                                                    | Determine the current Carbon grid breakpoint size |
+| Prop name | Required | Kind             | Reactive | Type                                         | Default value | Description |
+| :-------- | :------- | :--------------- | :------- | -------------------------------------------- | ------------- | ----------- |
+| sizes     | No       | <code>let</code> | Yes      | <code>Record<BreakpointSize, boolean></code> | <code>{       |
+
+     sm: false,
+     md: false,
+     lg: false,
+     xlg: false,
+     max: false,
+
+}</code> | Carbon grid sizes as an object |
```

Comment:
Was confused by all the changes here, looks like code sections are not generated correctly. Pretty sure stuff like this breaks the table.

Reply:
Could be a platform issue, `\r\n` vs just `\n` (i.e. Windows vs other systems).
In the JSON file there are now a bunch of `\r`, so the `\n` of an `\r\n` was probably stripped.

If this is correct, the issue might be in [sveld](https://github.com/carbon-design-system/sveld).

Reply:
Lemme know if there is anything I can do on my end to help resolve. I'm on a Win11 system using VSC.

As mentioned in a previous thread, I get heaps of changes in git when I do a rebuild of the docs. Could it be something to do with my configuration?

Reply:
If you have the Git setting [`core.autocrlf`](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf) on `true`, changing it to `false` could help (a new checkout may also be necessary to update the files). This flag causes files to be checked out with `\r\n` rather than just `\n`, in most cases it should not be necessary, though.

Of course ideally the docs would be generated correctly either way.